charts/fluent-bit/templates/servicemonitor.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
     {{- end }}
 spec:
+  jobLabel: {{ .Values.serviceMonitor.jobLabel | default .Release.Name }}
```

Comment:
Shouldn't this be conditional on the value being set rather than using a default?

Reply:
fixed as suggested